sccp/figures/figure3.py

```diff
@@ -37,7 +37,7 @@ def makeFigure():
     
     # UMAP dimension reduction
     umapReduc = umap.UMAP(random_state=1)
-    pf2Points = umapReduc.fit_transform(np.concatenate(projs, axis=0))
+    pf2Points = umapReduc.fit_transform(np.concatenate(projs, axis=0) @ factors[1])
```

Comment:
The UMAP positions should be based only on the projections, not B * projections which are the factor effects.